mautic/mautic
# Fix report asset downloads counts

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | 3.3
| Bug fix?                               | ✅ 
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | 
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 




#### Description:
This PR fix the count of assets downloads in reports in selected time frame.





#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create assets
3. Make download from two contacts - one download yesterday, second download today (two times)
4. Create report asset downloads with asset name, download count and unique download count
5. Select time frame for today
6. You should see count 3 downloads and  2 unique downloads on assets, but expected 2 downloads and 1 unique download for today
7. After fix should works properly
8. Try also reorder results by download count and download unique count
